docs/content/develop/make-a-breaking-change.md

```diff
@@ -181,6 +181,21 @@ Example (`google_container_cluster.enable_shielded_nodes`)
 * https://github.com/GoogleCloudPlatform/magic-modules/pull/5263 changed the default value for the field and added an upgrade guide entry
 * Note: In 4.0.0 the upgrade guide was split between main and the major release branch. Going forward, those changes will be made against main exclusively.
 
+## Contributing to the next major release (`5.0.0`)
+
+For the next major release, `5.0.0`, a specific branch named [`FEATURE-BRANCH-major-release-5.0.0`](https://github.com/GoogleCloudPlatform/magic-modules/tree/FEATURE-BRANCH-major-release-5.0.0) will be managed parallel to `main` in the magic-modules repo.
```

Comment:
Could we make this more user-focused? Along the lines of "For the `5.0.0` major release, the major release branch that you'll contribute to is [`FEATURE-BRANCH-major-release-5.0.0`](https://github.com/GoogleCloudPlatform/magic-modules/tree/FEATURE-BRANCH-major-release-5.0.0).`